demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -966,6 +969,52 @@ def _find_pack_dependencies(pack_id, id_set, verbose_file, exclude_ignored_depen
 
         return pack_dependencies
 
+    @staticmethod
+    def build_all_dependencies_graph(pack_ids, id_set, verbose_file, exclude_ignored_dependencies=True):
```

Comment:
why do we need `exclude_ignored_dependencies` arg? it is `True` by default in `_find_pack_dependencies` anyway

Reply:
What if someone wants to use the whole `build_all_dependencies_graph` method with `exclude_ignored_dependencies` as `False`?

Reply:
if you think it makes sense you can keep it, just dont see why would someone want to do so

Reply:
@yaakovi Any thoughts?